### PR TITLE
Allow the option of adding a specific config file

### PR DESCRIPTION
**Pull request by nicolasgrieco**
- As the main config.php is templated, options not handled by any attributes
   can be added in a second file in attribute.

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -250,14 +250,17 @@
...
   variables(
-    db_pass:  node['mariadb']['user_librenms']['password'],
-    user:     librenms_username,
-    path:     librenms_homedir,
-    auto_up:  node['librenms']['auto_update_enabled'],
-    xdp:      node['librenms']['autodiscover']['xdp'],
-    ospf:     node['librenms']['autodiscover']['ospf'],
-    bgp:      node['librenms']['autodiscover']['bgp'],
-    snmpscan: node['librenms']['autodiscover']['snmpscan'],
+<<<<<<< HEAD
```
Comment (olive42) [inline recipes/default.rb]: You have conflict markers in your file, you should fix that.

**Review by olive42 — CHANGES_REQUESTED** [review]

File: recipes/default.rb (hunk 1 above)
Comment (nicolasgrieco) [reply recipes/default.rb]: Fixed.

File: templates/default/config.php.erb [hunk 2]
```diff
@@ -47,3 +47,7 @@ $config['rrdtool_version'] = '1.5.5';
 
 $config['geoloc']['latlng'] = true;
 $config['geoloc']['engine'] = "google"; //Only one available at present
+
+<% if @add_conf_enabled == true %>
+ include "<%= @add_conf_file_path %>
```
Comment (cpaillet) [inline templates/default/config.php.erb]: can you check if the file exist ?
Can you add __DIR__ to the config file

`if(file_exists( __DIR__  . '<%= @add_conf_file_path %>')) {`
`    include  __DIR__  . '<%= @add_conf_file_path %>';`
`}`

File: templates/default/config.php.erb [hunk 3]
```diff
@@ -47,3 +47,9 @@ $config['rrdtool_version'] = '1.5.5';
 
 $config['geoloc']['latlng'] = true;
 $config['geoloc']['engine'] = "google"; //Only one available at present
+
+<% if @add_conf_enabled == true %>
+ if(file_exists( __DIR__ . '<%= @add_conf_file_path %>')) {
+ include "<%= @add_conf_file_path %>"
```
Comment (cpaillet) [inline templates/default/config.php.erb]: 4 spaces

**Review by cpaillet — COMMENTED** [review]
Identation

**Review by olive42 — APPROVED** [review]

**Pull request merged**